### DISCUSSION TITLE
Hot to get current content on [...slug].vue ?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/content#1542] [Q&A] [answered]
Hot to get current content on [...slug].vue ?

**Comment by nobkd** [comment]
Can you describe your problem/question in more detail?

**Reply by JenuelDev** [reply]
I mean where in the `[...slug[.vue` is it possible to get the current content like.
```js
const content = async content(); // something like this.

const title = content.title;
```

**Reply by nobkd** [reply, marked as answer]
I think you're looking for `useContent()`

You can see this for usage: https://github.com/nuxt/content/blob/main/src/runtime/pages/document-driven.vue

You can also look in the [api docs](https://content.nuxtjs.org/api/composables/use-document-driven.vue)

**Oh. Just seen *this* method only works in document driven mode**

When you're not using doc-driven mode, there should be another way

**Comment by ekkaiaaa** [comment]
Any update?
